Sources/Client/WebSocket/Provider/ProxyWebSocketProvider.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WebSocketProviderProxy.swift
+//  StreamChatClient
+//
+//  Created by Alexey Bukhtin on 06/05/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+final class ProxyWebSocketProvider: WebSocketProvider {
```

Comment:
This seems redundant. See https://getstream.slack.com/archives/CRB9P6L03/p1588842239493300